server/datastore/mysql/migrations/tables/20230719152138_ScheduledQueriesDataMigrator.go

```diff
@@ -0,0 +1,296 @@
+package tables
```

Comment:
We need to delete the Global and Team packs at the end too. (Something like `DELETE FROM packs WHERE pack_type = 'global' OR pack_type LIKE 'team-%'`.)